src/test/java/org/example/schoology/hooks/CourseHooks.java

```diff
@@ -40,4 +44,20 @@ public void deleteCourse() {
         // delete by Rest API (~3 milli seconds)
     }
 
+    @After(value = "@deleteResourceCollection")
+    public void deleteResourceCollection() {
+
+        DriverFactory.getDriver().get("https://app.schoology.com");
+        Login login = new Login();
+        Home home = login.loginAs(Environment.getInstance()
+                        .getValue(String.format("credentials.%s.username", "Instructor02")),
+                Environment.getInstance()
```

Comment:
este hook fallara si alguien usa un user distinto a Instructor02, tal vez podrias registrar en un context la informacion de usuario y el recurso para que sean eliminados

Reply:
Yes, this will be sent bt context injectionin my next branch code with Join Scenario groups